CHANGELOG.md

```diff
@@ -4,6 +4,55 @@
 New Features
 ------------
 
+- A RemoteParticipant now maintains a collection of RemoteTrackPublications in a
+  new `.trackPublications` property. It also maintains media specific
+  RemoteTrackPublication collections (`.audioTrackPublications`,
+  `.dataTrackPublications` and `.videoTrackPublications`). A RemoteTrackPublication
+  represents a Track that was published to the Room by a RemoteParticipant. A
```

Comment:
I don't know that we need to describe what a RemoteTrackPublication is again, because we already introduced it in a previous release, although only for the "subscriptionFailed" case, IIRC.